src/services/api.ts

```diff
@@ -530,6 +549,26 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             {concurrentId: concurrentId || 'getHotKeys', requestConfig: {signal}},
         );
     }
+
+    checkTrace({url}: {url: string}, {concurrentId, signal}: AxiosOptions = {}) {
+        return this.get<JsonHotKeysResponse>(
+            url,
+            {},
+            {
+                concurrentId: concurrentId || 'checkTrace',
+                requestConfig: {
+                    signal,
+                    timeout: TRACE_CHECK_TIMEOUT,
+                    'axios-retry': {
+                        shouldResetTimeout: true,
+                        retries: TRACE_CHECK_MAX_RETRIES,
+                        retryDelay: () => TRACE_CHECK_RETRY_DELAY,
+                        retryCondition: () => true,
```

Comment:
Should we always retry? Even if errored?

Reply:
Yes